### PR TITLE
feat(Pdpp): Overview screen

### PR DESCRIPTION
**Pull request by sigruntg**
# Overview Screen

https://app.clickup.com/t/j59jeq

## What

Adding overview screen to application. 
It is still missing functionality for what happens on edit. 
Need to add text into message file.

Also made changes to PaymentPlanTable so that it can be used in the Overview as well.
Added fields to the Table as well.

## Why

So that the user can see what was filled in the form on one page.

## Screenshots / Gifs

image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/public-debt-payment-plan/src/fields/Overview/Overview.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  // TODO: Add text to messages.
+  // TODO: Add edit functionality.
+  return (
+    <>
+      <ReviewGroup isEditable editAction={editAction}>
+        <GridRow>
+          <GridColumn span={['6/12', '5/12']}>
+            <Box>
+              <Label>Nafn</Label>
+              <Text>Fjármundur Skuldason</Text>
```
Comment (andriandresson) [inline libs/application/templates/public-debt-payment-plan/src/fields/Overview/Overview.tsx:28]: Shouldn't this be provided by the application.answers ?

File: libs/application/templates/public-debt-payment-plan/src/fields/PaymentPlan/PaymentPlan.tsx [hunk 2]
```diff
@@ -180,7 +185,18 @@ export const PaymentPlan = ({ application, field }: FieldBaseProps) => {
         />
       )}
       {(isLoading || paymentPlanResults) && (
```
Comment (andriandresson) [inline libs/application/templates/public-debt-payment-plan/src/fields/PaymentPlan/PaymentPlan.tsx:187]: Do we need this check ? 
Looks like this is being handled in the PaymentPlanTable component

**Review by andriandresson — COMMENTED** [review]
Few comments

File: libs/application/templates/public-debt-payment-plan/src/fields/Overview/Overview.tsx (hunk 1 above)
Comment (sigruntg) [reply libs/application/templates/public-debt-payment-plan/src/fields/Overview/Overview.tsx:28]: Yup, that's why it's in the TODO above. This is not completely finished.

File: libs/application/templates/public-debt-payment-plan/src/fields/PaymentPlan/PaymentPlan.tsx (hunk 2 above)
Comment (sigruntg) [reply libs/application/templates/public-debt-payment-plan/src/fields/PaymentPlan/PaymentPlan.tsx:187]: That's true. I will add this to the clickup card. I don't have time to remove it now. But would be great to get this live to get some sexy fixes.

File: libs/application/templates/public-debt-payment-plan/src/fields/PaymentPlan/PaymentPlan.tsx [hunk 3]
```diff
@@ -1,6 +1,12 @@
 import { FieldBaseProps } from '@island.is/application/core'
 import { useAuth } from '@island.is/auth/react'
-import { Box, RadioButton, Stack, Text } from '@island.is/island-ui/core'
+import {
+  AccordionItem,
```
Comment (Thorrik58) [inline libs/application/templates/public-debt-payment-plan/src/fields/PaymentPlan/PaymentPlan.tsx:4]: unused

**Review by Thorrik58 — APPROVED** [review]
LGTM with the prefix that its still a WIP

**Pull request merged**